ios/ble/central/Central.swift

```diff
@@ -70,4 +68,13 @@ class Central: NSObject, CBCentralManagerDelegate {
             connectedPeripheral.writeValue(messageData, for: characteristic, type: .withoutResponse)
         }
     }
+    
+    func onDeviceDisconnected(isManualDisconnect: Bool) {
```

Comment:
This exact function already exists on Wallet.swift file, why is it needed here again?
Also, the function signature seems to be different from Android implementation.

Reply:
Removed the function in wallet. added because of merge